packages/common/src/sites/_internal/SiteBusinessRules.ts

```diff
@@ -121,6 +128,36 @@ export const SitesPermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:site:discussions",
     dependencies: ["hub:site:view"],
   },
+  {
+    permission: "hub:site:workspace:overview",
+    dependencies: ["hub:site:view"],
+  },
+  {
+    permission: "hub:site:workspace:dashboard",
+    dependencies: ["hub:site:view"],
+  },
+  {
+    permission: "hub:site:workspace:details",
+    dependencies: ["hub:site:edit"],
+  },
+  {
+    permission: "hub:site:workspace:settings",
+    dependencies: ["hub:site:edit"],
+    entityOwner: true,
+  },
+  {
+    permission: "hub:site:workspace:collaborators",
+    dependencies: ["hub:site:edit"],
+  },
+  {
+    permission: "hub:site:workspace:content",
+    dependencies: ["hub:site:edit"],
+  },
+  {
+    permission: "hub:site:workspace:metrics",
+    dependencies: ["hub:site:edit"],
+    environments: ["devext", "qaext"],
```

Comment:
@sonofflynn89 and @abp6318 - we should be following this pattern if we need gate content workspace panes

Reply:
fwiw - I'll be adding these permissions into the <Entity>WorkspaceLinks files in opendata-ui as a follow-up PR - so this  initial set of things will be there later today